jsonnet/components/prometheus.libsonnet

```diff
@@ -382,6 +382,10 @@ function(params)
             value: '15ms',
           },
         ],
+        // Increase the startup probe timeout to 1h from 15m to avoid restart
+        // failures when the WAL replay takes a long time.
+        // See https://issues.redhat.com/browse/OCPBUGS-4168 for details.
+        maximumStartupDurationSeconds: 3600,
```

Comment:
According to https://github.com/prometheus-operator/prometheus-operator/blob/e0b26a167be886fa2a8cce24109bb5599dce6728/pkg/prometheus/statefulset.go#L504

We'll end up with a `PeriodSecond` of 60s, this maybe make prometheus take 45s longer to be ready. Is it ok?

Reply:
That's correct. I feel like it's an acceptable trade-off. The (indirect) upside is that it will leave more time for the newly started Prometheus pod to scrape targets so it minimizes the risk of metric gaps.